monkey/infection_monkey/exploit/sshexec.py

```diff
@@ -156,7 +193,9 @@ def _exploit_host(self) -> ExploiterResultData:
 
         try:
             with ssh.open_sftp() as ftp:
```

Comment:
The default tiemout of this is 1h. This might be a problem

Reply:
Can you do a test? Set a breakpoint at this line, kill the victim and step over. See how long it takes for this line to execute

Reply:
open_sftp doesn't seem to have timeout .... 

![image](https://user-images.githubusercontent.com/15820737/159014566-63517a08-740f-4861-b249-0e41d4c0f8e1.png)

Reply:
Actually you are right. If a machine gets killed after we connect, `open_sftp` will wait one hour for a connection and block.